Kronos-Shifts-Connector/Microsoft.Teams.Shifts.Integration/.editorconfig

```diff
@@ -8,6 +8,3 @@ dotnet_style_qualification_for_method = true:silent
 
 # CA1724: The type name OpenShiftRequest conflicts in whole or in part with the namespace name 'Microsoft.Teams.App.KronosWfc.Models.RequestEntities.OpenShift.OpenShiftRequest'. Change either name to eliminate the conflict.
 dotnet_diagnostic.CA1724.severity = warning
-
-# CA1303: Do not pass literals as localized parameters
```

Comment:
Why did you remove this

Reply:
@aosolis  It was removed erroneously through merging master on to your branch. Restored the same.